src/components/Icons/Icon.tsx

```diff
@@ -15,12 +15,62 @@ const StyledFontAwesomeIcon = styled(FontAwesomeIcon)`
     width: 1em;
     height: 1em;
     margin: 0.125em;
-    color: ${(props) => props.color};
+    color: ${(props) => props.color ?? props.theme.accentBackgroundColor};
+`;
+
+const Popper = styled.div`
+    position: absolute;
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
+    color: ${(props) => props.theme.surfaceForegroundColor};
+    padding: 0.5em;
+    border-radius: 0.5em;
+    z-index: 10;
+    font-size: 0.7em;
+
+    transform: translateY(2em);
+
+    box-shadow: 0.05rem 0.05rem 0.2rem ${(props) => props.theme.disabledColor};
 `;
 
 const Icon: React.FC<Props> = (props) => {
+    const [hovered, setHovered] = React.useState(false);
+
+    const show = (): void => {
+        setHovered(true);
+    };
+
+    const hide = (): void => {
+        setHovered(false);
+    };
+
     return (
-        <StyledFontAwesomeIcon icon={props.icon} title={props.onHoverText} color={props.color} />
+        <>
+            {props.onHoverText && hovered ? (
+                <Popper
+                    onMouseEnter={show}
+                    onMouseLeave={hide}
+                    onMouseDown={show}
+                    onMouseUp={hide}
+                    onTouchStart={show}
```

Comment:
is it worth having the `touch` mobile events be toggles on any tap, rather than just while long holding? I know my iPhone will immediately go to a 'different view' if you long hold for too long.

(also might be better to have multiple open at the same time! might want to just mess around with different approaches and see what 'feels' most natural 🙂)